### PR TITLE
docs(contributing): Visual review standards

### PR DESCRIPTION
**Pull request by Droogans**
@glynnis @chriscantu feel free to simply push over these commits on my fork with anything that you need to see here.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Sometimes changes are urgent, and in this case, the PR should be reviewed immedi
 - Complete documentation (a docs subfolder with working examples and [ngdocs](https://github.com/angular/angular.js/wiki/Writing-AngularJS-Documentation)), along with inline code comments as beneficial
 - Unit tests with 80% line coverage
 - Midways tests for all new UI functionality
+- Screenshots highlighting design changes/proposals
```
Comment (glynnis) [inline CONTRIBUTING.md]: ~~/proposals~~

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -108,6 +109,35 @@ Once a pull request has been submitted, you simply need to wait for the EncoreUI
 
 We like to at least comment on, if not accept, pull requests within three business days (and, typically, one business day). We may suggest some changes or improvements or alternatives, so **make sure there is time for review in your release plan**.
 
+### Design Review of Pull Requests
+
+Many times, pull requests will touch resources that render visually. These are still changes that must be reviewed! For the sake of simplicity and transparency, it is the responsibility of the author of a pull request to also include a comment containing screenshots of the visual changes. Here is an example of a typical pull request workflow containing designer feedback and sign off.
+
+1. A pull request author finishes some new feature that contains visual changes.
```
Comment (glynnis) [inline CONTRIBUTING.md]: "that contains visual changes already mocked-up and approved by design."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -108,6 +109,35 @@ Once a pull request has been submitted, you simply need to wait for the EncoreUI
...
+0. The pull request author gathers the appropriate before (master branch, usually) and after screenshots of each relevant section.
+0. A pull request is created.
+0. A comment is created [that tags an Encore-UI designer](https://github.com/blog/821). Immediately below this "@" mention, the screenshots are [uploaded in the comment interface](https://help.github.com/articles/issue-attachments/).
+0. The initial visual feedback suggestion from our team recommends changing something. This change would require new screenshots!
```
Comment (glynnis) [inline CONTRIBUTING.md]: "The initial visual feedback suggestion from our design team recommends changing something so it better complies with original mock-ups they provided."

File: CONTRIBUTING.md (hunk 1 above)
Comment (Droogans) [inline CONTRIBUTING.md]: > Screenshots highlighting design implementation

?

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**